### PR TITLE
Use the API endpoint to store a new document when submitting the create document form

### PR DESCRIPTION
**Pull request by hackermariama**

### CODE REVIEW

File: client/src/api/documents.js [hunk 1]
```diff
@@ -7,22 +7,35 @@ export const getDocuments = () => {
...
+      due_date,
+      content
+    }),
+    headers: {
+      Authorization: `Bearer ${sessionStorage.getItem("token")}`,
+      "Content-Type": "application/json"
+    }
+  })
+    .then(res => res.json())
+    .then(this.handleErrors)
```
Comment (llh1) [inline client/src/api/documents.js]: This will not work as this.handleErrors is not defined here.

File: client/src/api/documents.js [hunk 2]
```diff
@@ -7,22 +7,35 @@ export const getDocuments = () => {
...
+    .then(this.resetForm)
```
Comment (llh1) [inline client/src/api/documents.js]: This will not work as this.resetForm is not defined here.

File: client/src/api/documents.js [hunk 3]
```diff
@@ -7,22 +7,35 @@ export const getDocuments = () => {
...
+    .catch(error => {
+      this.setState({
```
Comment (llh1) [inline client/src/api/documents.js]: You can only use setState inside a React component. You need to do all of this in a React component.

**Review by llh1 — CHANGES_REQUESTED** [review]

File: client/src/api/documents.js (hunk 1 above)
Comment (hackermariama) [reply client/src/api/documents.js]: its defined now

File: client/src/api/documents.js (hunk 2 above)
Comment (hackermariama) [reply client/src/api/documents.js]: its fixed now

File: client/src/api/documents.js (hunk 3 above)
Comment (hackermariama) [reply client/src/api/documents.js]: done

**Review by llh1 — APPROVED** [review]

**Pull request merged**